class-debug-bar-cron.php

```diff
@@ -380,6 +381,17 @@ private function display_schedules() {
 				</table>';
 		}
 
+		/**
+		 * Sorting method for cron scheldules. Order by scheldule interval
+		 *
+		 * @param array $a first element of compasion  pair
+		 * @param array $b second element of compasion  pair
+		 *
+		 * @return int 1 if element $a interval greater then element $b interval, -1  otherwise.
+		 */
+		function schedules_sorting($a, $b){
+			return (int) $a['interval'] > $b['interval'];
```

Comment:
This does not what it's supposed to do.

`(int) $a['interval'] > $b['interval']` will return either true or false, not 1 or -1.

Similarly if you want to do the comparison numerically, you should cast both values to `int`:
`( (int) $a['interval'] > (int) $b['interval'] )`